docs/src/introduction.md

```diff
@@ -1,24 +1,39 @@
-# Polygon Miden
+# Polygon Miden - Privacy scales better
+Miden is a zero-knowledge rollup for high-throughput and private applications. Miden allows users to prove state changes of local data where the network only tracks a commitment of. Additionally, users can also let the Operator prove public state changes as in other known rollups.  
```

Comment:
A few comments:

1. I would probably remove `Privacy scales better" from the title. If we want to make it very visible, we could use it as quote in this section.
2. We should have at least one sentence touching on "Privacy scales better" here. Otherwise we say it very prominently, but don't explain what it means.
3. nit: "Additionally, ... also" sounds redundant in the last sentence.

A potential bigger change for the future. I would maybe replace the last 2 sentences here and a part of the following paragraph with a bullet-point list of Miden's goals or something like that. But we should align on this list first.

Reply:
thanks, changed all of the above